src/advanced/merkelized-list.md

```diff
@@ -137,7 +150,7 @@ If `height > 1`, node `T(height - 1, index * 2)` exists and
 node `(height - 1, index * 2 + 1)` is absent in the tree, then:
 
 ```none
-T(height > 1, index) = hash(T(height - 1, index * 2)).
+T(height > 1, index) = hash(0x1 || T(height - 1, index * 2)).
```

Comment:
Nit: I know this was here before, but does it make sense to specify `> 1` on the LHS of the formula? IMO, it doesn't.